### PR TITLE
adding: home page to video library

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
In this PR request add Home Page for the Video library.

* use this link to see the files that need to be reviewed  ▶️ [Review me only](https://github.com/santhoshpemmaka/NamasteVideo/pull/1/files?file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true).

image
image

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			<footer className='footer footer-align'>
+				<div className='footer-name'>
+					Made with <span className='footer-symbol'>&lt;/&gt;</span> by Santhosh
+					Pemmaka
+				</div>
+				{footerItems && (
+					<ul className='ul-tag-footer ul-footer'>
+						{footerItems &&
+							footerItems.length > 0 &&
+							footerItems.map((footer_item) => (
```
Comment (iRohitGaur) [inline src/components/Footer/Footer.jsx:30]: suggestion: on #28 you can remove `footerItems &&` as you already checked that on #26

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+// import {useAuthentication} from "../../../Context/AuthContext";
+const Header = () => {
+	const [showNav, setshowNav] = useState(false);
+	// const {state} = useStateContext();
+	// const {itemInCart, itemInWishList} = state;
+	// const navigate = useNavigate();
+	// const {
+	// 	state: {token, userName},
+	// 	logoutUser,
+	// } = useAuthentication();
```
Comment (iRohitGaur) [inline src/components/Header/Header.jsx:14]: suggestion: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by dev-prabhat — APPROVED** [review]
Just remove commented code and work on the given suggestion

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (santhoshpemmaka) [reply src/components/Footer/Footer.jsx:30]: I got it. I made changes. thanks.

File: src/components/Header/Header.jsx (hunk 2 above)
Comment (santhoshpemmaka) [reply src/components/Header/Header.jsx:14]: Comment lines used auth feature. I removed next pr request.

**Pull request merged**